.github/workflows/push_hub.yml

```diff
@@ -6,6 +6,8 @@ on:
   pull_request:
     branches:
     - master
+  schedule:
+    - cron: "24 04 1 * *"
```

Comment:
This probably was used for testing and is no longer needed?

Reply:
No not really, this ensures that this particular action is run every first day of the month (04:24). Thus it ensures that the docker image is rebuild every month even when there are no commits. This ensures that every month, a new docker image is created with the latest security patches from the Debian base image.

Reply:
Oh, gotcha :thinking: 

Then I'll guess @msm-code will have to look at it